src/lib/sentry-sanitizer.ts

```diff
@@ -0,0 +1,137 @@
+import { Blueprint } from '@wp-playground/blueprints';
+import { RunCLIArgs } from '@wp-playground/cli';
+
+/**
+ * Sanitizes a Blueprint step to remove sensitive data while keeping useful debugging info.
+ */
+function sanitizeBlueprintStep( step: Blueprint[ 'steps' ][ number ] ): Record< string, unknown > {
+	const baseStep: Record< string, unknown > = { step: step.step };
+	const stepRecord = step as Record< string, unknown >;
+
+	// For steps that might contain secrets, only include safe fields
+	switch ( step.step ) {
+		case 'login':
+			// Omit login details, indicate it exists
+			return { ...baseStep, hasLogin: true };
+
+		case 'runPHP':
+		case 'runPHPWithOptions':
+			// Omit code (might contain secrets), indicate it exists
+			return { ...baseStep, hasCode: !! stepRecord.code };
+
+		case 'defineWpConfigConsts':
+			// Omit actual constants (might contain DB credentials, auth keys)
+			return {
+				...baseStep,
+				constCount:
+					stepRecord.consts && typeof stepRecord.consts === 'object'
+						? Object.keys( stepRecord.consts as object ).length
+						: 0,
+			};
+
+		case 'runSql':
+			// Omit SQL (might contain sensitive data)
+			return { ...baseStep, hasSql: !! stepRecord.sql };
+
+		case 'writeFile':
+			// Keep path for debugging, omit file content
+			return {
+				...baseStep,
+				path: stepRecord.path,
+				hasData: !! stepRecord.data,
+			};
+
+		case 'request':
+			// Keep URL and method, omit headers and body (might contain auth tokens)
+			return {
+				...baseStep,
+				url:
+					stepRecord.request && typeof stepRecord.request === 'object'
+						? ( stepRecord.request as Record< string, unknown > ).url
+						: undefined,
+				method:
+					stepRecord.request && typeof stepRecord.request === 'object'
+						? ( stepRecord.request as Record< string, unknown > ).method
+						: undefined,
+			};
+
+		case 'setSiteOptions':
+		case 'updateUserMeta':
+			// Keep option/meta keys but not values (values might be API keys)
+			return {
+				...baseStep,
+				keys:
+					stepRecord.options && typeof stepRecord.options === 'object'
+						? Object.keys( stepRecord.options as object )
+						: stepRecord.meta && typeof stepRecord.meta === 'object'
+						? Object.keys( stepRecord.meta as object )
+						: [],
+			};
```

Comment:
```suggestion
		case 'updateUserMeta': {
			// Keep option/meta keys but not values (values might be API keys)
			let keys: string[] = [];
			if ( stepRecord.options && typeof stepRecord.options === 'object' ) {
				keys = Object.keys( stepRecord.options as object );
			} else if ( stepRecord.meta && typeof stepRecord.meta === 'object' ) {
				keys = Object.keys( stepRecord.meta as object );
			}

			return {
				...baseStep,
				keys,
			};
		}
```
Nit, but let's unwrap the nested ternary.